pkg/R/RDD.R

```diff
@@ -23,7 +24,10 @@ setClass("PipelinedRDD",
 
 
 setMethod("initialize", "RDD", function(.Object, jrdd, serialized,
```

Comment:
At the top of this method we should add a comment saying what the accepted values are for `serialized` and what they mean